doc/whats_new.rst

```diff
@@ -31,6 +31,8 @@ Changelog
 
     - Add support for the University of Maryland KIT system by `Christian Brodbeck`_
 
+    - Add the `--filterchpi` option to `mne browse_raw`, by `Felix Raimundo`_
```

Comment:
double-backticks on the filterchpi and mne browse_raw because the links won't resolve properly as-is

bonus points if you want to dig into our doc build and figure out how to link to the command page

@agramfort any idea where our Python command-line functions get documented? We could probably figure out the links from there